pkg/component/extensions/extension/extension_test.go

```diff
@@ -322,9 +322,8 @@ var _ = Describe("Extension", func() {
 				time.Microsecond*400,
 				time.Second,
 			)
-			Expect(ext.DestroyBeforeKubeAPIServer(ctx)).To(MatchError(&multierror.Error{
-				Errors: []error{fakeError, fakeError},
-			}))
+			var expectedErr error = &multierror.Error{Errors: []error{fakeError, fakeError}}
+			Expect(ext.DestroyBeforeKubeAPIServer(ctx)).To(MatchError(expectedErr))
```

Comment:
```suggestion
			Expect(ext.DestroyBeforeKubeAPIServer(ctx)).To(MatchError(&multierror.Error{
				Errors: []error{fakeError, fakeError},
			}.(error)))
```

Would this work as well? If yes, I'd prefer this style instead of declaring a separate variable.

Reply:
Nope because `variable.(error)` returns 2 values.
```
Expect(ext.DestroyBeforeKubeAPIServer(ctx)).To(MatchError(error(&multierror.Error{Errors: []error{fakeError, fakeError}})))
```
is working.